<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng nhập - Employee Management System</string>
  </property>
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>ĐĂNG NHẬP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-weight: bold; font-size: 16pt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblEmail">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtEmail">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Nhập email của bạn</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblPassword">
       <property name="text">
        <string>Mật khẩu:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtPassword">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Nhập mật khẩu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnLogin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Đăng nhập</string>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #9CAFAA;
    color: white;
    border: none;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #7A9A8A;
}
QPushButton:pressed {
    background-color: #6A8A7A;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Hủy</string>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #CCCCCC;
    color: black;
    border: none;
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #BBBBBB;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblAttempts">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
     </property>
     <property name="styleSheet">
      <string>color: red; font-size: 10pt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

